app/src/lib/editor/engine/projectinfo/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { ReactComponentDescriptor } from '/electron/main/code/components';
+
+export class ProjectInfoManager {
+    private projectComponents: ReactComponentDescriptor[];
+    constructor() {
+        this.projectComponents = [];
```

Comment:
This should call `makeAutoObservable(this, {}); ` to be subscribable

Reply:
fixed in https://github.com/onlook-dev/onlook/pull/285/commits/fee273ac862375ba4503c3ae5844f5f58e1eb009